src/Makefile.am

```diff
@@ -205,7 +207,7 @@ libanoncoin_wallet_a_SOURCES = \
   $(ANONCOIN_CORE_H)
 
 # crypto primitives library
-crypto_libanoncoin_crypto_a_CPPFLAGS = $(ANONCOIN_CONFIG_INCLUDES)
+crypto_libanoncoin_crypto_a_CPPFLAGS = $(ANONCOIN_CONFIG_INCLUDES) -DENABLE_I2PD -I/usr/local/Cellar/openssl/1.0.2o_1/include
```

Comment:
either bundle this, or include build for it. including such things into the build system sucks big...

Reply:
Yes we don't merge this ever - just for reference for the #23